go-controller/pkg/node/gateway_localnet.go

```diff
@@ -406,6 +406,9 @@ func getLoadBalancerIPTRules(svc *kapi.Service, svcPort kapi.ServicePort, gatewa
 //    the return traffic can be steered back to OVN logical topology
 // -- to also handle unDNAT return traffic back out of the host
 func newLocalGatewayOpenflowManager(patchPort, macAddress, gwBridge, gwIntf string) (*openflowManager, error) {
+	// 14 bytes of overhead for ethernet header (does not include VLAN)
+	maxPktLength := config.Default.MTU + 14
```

Comment:
@trozet any way we can consolidate this into a function so we don't have it in two places (here and shared code)?

Reply:
Looks like all three places use the same code for 'actions' and 'dftFlows' too?

Reply:
sure will do

Reply:
i moved the max pkt length calculation to its own function, but I think im going to keep the flows explicitly where they are. I just feel like even though theres a small amount of duplication here, it makes it more obvious what flows are being installed. If you strongly disagree then I'll change it.